docs/generated/packages/angular.json

```diff
@@ -228,6 +228,11 @@
             "description": "Generate an application that is setup to use standalone components.",
             "type": "boolean",
             "default": false
+          },
+          "skipNXWelcomeComponent": {
```

Comment:
`NX` is spelled `Nx`. See: https://nx.dev/getting-started/intro

So maybe change that as well?

Reply:
Updated to Nx.